docs/DeveloperGuide.adoc

```diff
@@ -297,6 +297,7 @@ The undo/redo mechanism is facilitated by an `UndoRedoStack`, which resides insi
 
 `UndoRedoStack` only deals with `UndoableCommands`. Commands that cannot be undone will inherit from `Command` instead. The following diagram shows the inheritance diagram for commands:
 
+.Logic Class Diagram
```

Comment:
I like that you have labelled the diagram, we will continue using this format for the rest of the DG.

@codeeong @lithiumlkid do follow this format since it's already been updated for the rest of the DG.

Reply:
Yup i labelled the diagrams in front too! @lithiumlkid @codeeong